ci/linux/create_venv.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env sh
 
-python3 -m venv ./.venv
+python3.11 -m venv ./.venv
```

Comment:
This ties the development environment to 3.11 for everyone. Is there a reason we need 3.11?

Reply:
Cheers for https://github.com/Project-OMOTES/omotes-sdk-protocol/pull/14/commits/07bcdb199458e4b3024f5bc9d7981a9bfd151de2 but now it is tied to python 3.8. Is there a reason a specific Python version is required? Originally it was `python3` which used any available python 3 version.

Reply:
Isn't it better to fix the python version for consistency?

Reply:
Discussed it outside of Github. MarkTNO was right that it would increase consistency. Python version can cause dependencies to behave differently as a different python version may lead to a different version of a dependency to be installed. We chose to restrict this repository to Python 3.11 to ensure the protoc mypy output is always the same.